src/runtime_src/core/include/xcl_graph.h

```diff
@@ -89,4 +89,48 @@ xclStopProfiling(xclDeviceHandle handle, int phdl);
 int
```

Comment:
Copyright needs to be updated (line 3).  Being that we are part of AMD now, the following addition should be made.
`Copyright (C) Advanced Micro Devices, Inc.`